### DISCUSSION TITLE
天使动漫 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#135] [模板讨论]
天使动漫 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by Franinc** [comment]
`www.tsdm39.net`被sni rst了，两个签到可换到`www.tsdm39.com`

另外问下6小时间隔的打工签到，日志交替显示`成功领取了奖励天使币`和`不要作弊`，正常吗？还是需要我把间隔时间拉长，比如间隔6小时5秒/6小时1分？

**Reply by wjf0214** [reply]
天使动漫更新了，域名已经替换。
间隔6小时整那是会这样。执行还需要一定时间，到6小时了。实际6小时间隔肯定还没到，需要6小时多点。框架定时没法这么精准，所以保底一天3次还是可以的。想要6小时时间一到就执行，最好还是脚本的方式执行。

**Reply by Franinc** [reply]
试了下延迟拉到8小时还是循环`成功领取天使币`和`不要作弊`，`不要作弊`后检查论坛天使币也确实没增加。
索性延迟改回6小时，把`不要作弊`加到失败条件，6小时15分钟内能完成一次（默认失败重试延迟10分钟）

**Reply by wjf0214** [reply]
![image](https://user-images.githubusercontent.com/23102821/234155493-07e2eb23-109a-4753-b9bc-fd59bae125a5.png)
我目前OK。我就一天三次。。算是7个小时一次吧。

**Reply by Franinc** [reply]
不好意思是版本旧了没更新的问题。删了重下后正常了。间隔6小时2分钟

**Comment by yushizui** [comment]
签到失败，好像因为在https://www.tsdm39.com/forum.php 用name="formhash" value="(.+?)"提取失败，可以改成formhash=(.+?)"成功

**Comment by miaoxiaozhao** [comment]
Failed at 2/4 request,
Fail assert: {"re": "未定义操作", "from": "content"} from failed_asserts,
Request URL: https://www.tsdm39.com/plugin.php?id=dsu_paulsign%3Asign&operation=qiandao&infloat=1&sign_as=1&inajax=1
